tool/plugin/lib/plugin.dart

```diff
@@ -1148,3 +1147,35 @@ class TestCommand extends ProductCommand {
     throw 'integration test execution not yet implemented';
   }
 }
+
+Future<String> makeDevLog(BuildSpec spec) async {
+  _checkGitDir();
+  var gitDir = await GitDir.fromExisting(rootPath);
+  var since = await lastRelease();
+  var processResult =
+      await gitDir.runCommand(['log', '$since..HEAD', '--oneline']);
```

Comment:
Can you include some sample output from this? There are some options you can pass to `log` to tweak the output.

Reply:
I could make it check for capitalization of the first word, but haven't gotten that far.
```
## Changes since release_42
- if track widget creation is not enabled, pass in --no-track-widget-creation (#4196)
- Surface reload exceptions (#4198)
- provide more stack trace details for inspector timeouts (#4193)
- Update change log for M42.1 (#4192)
```

Reply:
Thanks; sounds like we can tweak the output here as necessary.